oauth2/oauth2-client/src/test/java/org/springframework/security/oauth2/client/registration/ClientRegistrationsTest.java

```diff
@@ -168,6 +168,30 @@ private void assertIssuerMetadata(ClientRegistration registration,
 				"grant_types_supported", "token_endpoint", "token_endpoint_auth_methods_supported", "userinfo_endpoint");
 	}
 
+	@Test
```

Comment:
To conform with other bug resolutions, would you please add a comment above each of these tests indicating the ticket that it resolves?

```suggestion
	// gh-7512
	@Test
```

Reply:
👍